### PR TITLE
[FEATURE] Add dashboard widget for quick access of the project documentation

### PR DESCRIPTION
**Pull request by peterkraume**

### CODE REVIEW

**Comment by georgringer** [comment]
@peterkraume does this work when dashboard is not installed?

**Comment by peterkraume** [comment]
Good point. Unfortunately currently it doesn't work without dashboard being installed. I'll get in touch with Richard Haeser.

File: Configuration/Services.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+services:
```
Comment (haassie) [inline Configuration/Services.yaml:1]: Please use the PHP version of the config to check if EXT:dashboard is installed before registering the widgets. An example can be found on https://github.com/FriendsOfTYPO3/widgets/blob/master/Configuration/Services.php

**Comment by peterkraume** [comment]
@haassie I've switched from Services.yaml to Services.php now and for me it's working now as expected:
- EXT:dashboad not installed: nothing happens, the configuration is ignored
- EXT:dashboad **is** installed: the new dashboard widget is available

Thx a lot for your hint. Maybe you can review my patch as well.

**Review by georgringer — APPROVED** [review]

**Pull request merged**